src/main/java/net/glowstone/GlowWorld.java

```diff
@@ -1,5 +1,6 @@
 package net.glowstone;
 
+import io.netty.util.internal.ConcurrentSet;
```

Comment:
"It works" should not be the quality we aim for. Look at package-info.java in Netty.